### DISCUSSION TITLE
ReEDS to PLEXOS linkage

### DISCUSSION THREAD

**Discussion by keurek** [NREL/ReEDS-2.0#37] [Q&A: Features] [answered]
There are several publications that reference a tool to link ReEDS with PLEXOS.
* ReEDS v2019 documentation - https://www.nrel.gov/docs/fy20osti/74111.pdf
* NREL 2020 Standard Scenarios - https://www.nrel.gov/docs/fy21osti/77442.pdf
* Frew et al (2019), see the SI - https://www.sciencedirect.com/science/article/pii/S2589004219303967

Is the linking tool publicly available? Is there detailed documentation about the linkage?

**Comment by keurek** [comment]
While ReEDS is publicly available through the Open Access GitHub repository, the ReEDS/PLEXOS linkage is not. 

We do not have an official documentation for the ReEDS/PLEXOS linkage as this time.

**Comment by pz-max** [comment]
@keurek, are there any updates on this thread? Is there something public now?

**Comment by pesap** [comment, marked as answer]
The ReEDS2Plexos linkage was renamed to R2X and currently lives on this repository: https://github.com/NREL/R2X
